docs/source/tutorials/datasets.rst

```diff
@@ -0,0 +1,457 @@
+
+
+..
+    TODO: Some things are missing here:
+        1. A link to where cloud integrations are enabled on the web-app
+        2. A way to enable cloud integrations from the SDK?
+        3. Sample outputs of the given script
+
+
+
+Deleting data
+-------------
+
+You can remove both Videos and Image Groups from datasets created using both the |platform| and the |sdk|.
+Use the method :meth:`dataset_client.delete_data() <.EncordClientDataset.delete_data>` to delete from a dataset.
+
+.. autolink-concat:: section
+
+.. autolink-preface::
+
+    from encord.client import EncordClientDataset
+    dataset_client = EncordClientDataset()
+
+.. code-block:: python
+
+    dataset_client.delete_data(
+        [
+            "<video1_data_hash>",
+            "<image_group1_data_hash>",
+        ]
+    )
+
+
+In case the Video or Image Group belongs to |company|-hosted storage, the corresponding file shall be removed from the Encord-hosted storage.
+
+Please ensure that the list contains Videos/Image Groups from the same dataset which is used to initialise the :class:`dataset_client <.EncordClientDataset>`.
+Any Videos or Image Groups which do not belong to the dataset used for initialisation would be ignored.
+
+
+Re-encoding videos
+------------------
+
+As videos come with various formats, frame rates, etc., one may in rare cases experience some frame-syncing issues on the |platform|.
+Specifically, it can be the case that, e.g., frame 100 on the |platform| does not correspond to the hundredth frame that you load with python.
+A browser test in the |platform| can tell you if you are at risk of experiencing this issue.
+
+.. TODO: we desperately need a link to documentation here ^
+
+To mitigate such issues, you can re-encode your videos to get a new version of your videos that do not exhibit such issues.
+
+Trigger a re-encoding task
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You re-encode a list of Videos by triggering a task for the same using the |sdk|.
+Use the method :meth:`dataset_client.re_encode_data() <.EncordClientDataset.re_encode_data>` to re-encode the list of videos specified
+
+.. tabs::
+
+    .. tab:: Code
+
+        .. code-block:: python
+
+            task_id = dataset_client.re_encode_data(
+                [
+                    "video1_data_hash",
+                    "video2_data_hash",
+                ]
+            )
+            print(task_id)
+
+    .. tab:: Output example
+
+        .. code-block:: python
+
+            1337   # Some integer
+
+
+On completion, a ``task_id`` is returned which can be used for monitoring the progress of the task.
+
+Please ensure that the list contains Videos from the same dataset which is used to initialise the EncordClient.
+Any Videos which do not belong to the dataset used for initialisation would be ignored.
+
+
+Check the status of a re-encoding task
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can monitor the status of an existing Re-encoding Task using this |sdk|.
```

Comment:
I would delete this line and concatenate it with the next to make it more succinct.